machine-learning/solutions/kubeflow-pipelines-sentiment-analysis/sentiment-analysis/pom.xml

```diff
@@ -0,0 +1,481 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This pom.xml is much too broad for this use case (multiple runners etc). We should simplify this. Can you use this pom.xml [1] and just add/remove additional dependencies that you need. This pom.xml should not have multiple runners and profiles so should be much simpler:

[1] https://github.com/GoogleCloudPlatform/professional-services/blob/master/examples/dataflow-bigquery-transpose/pom.xml